src/main/java/org/kairosdb/core/http/rest/metrics/DefaultQueryMeasurementProvider.java

```diff
@@ -108,13 +112,15 @@ private void measureSpan(final Histogram histogram, final QueryMetric query) {
         final long spanInMillis = endTime - query.getStartTime();
         final long spanInMinutes = spanInMillis / 1000 / 60;
         histogram.update(spanInMinutes);
+        tracer.activeSpan().setTag("query_span_in_days", spanInMinutes/1440);
     }
 
     private void measureDistance(final Histogram histogram, final QueryMetric query) {
         final DateTime nowUTC = new DateTime(DateTimeZone.UTC);
         final long distanceInMillis = nowUTC.getMillis() - query.getStartTime();
         final long distanceInMinutes = distanceInMillis / 1000 / 60;
         histogram.update(distanceInMinutes);
+        tracer.activeSpan().setTag("query_distance_in_days", distanceInMinutes/1440);
```

Comment:
probably the same problem as with row_count, this should be in the log IMO

Reply:
Currently Lightstep UI is not searchable by log. So it makes sense to have it as a tag which can be used to search for spans.

Reply:
How can you search counts?

p.s. let's keep vendor specific details away from OSS PRs.

Reply:
It is likely possible to have better search queries in future or with change of tracing backend. So do we make the changes in the instrumentation again? 
May be it makes sense to create buckets of counts based on the initial experience. The search works for me as you can filter down tag values through autocomplete.

Reply:
>Actually this argument is in favor of instrumentation assuming least capabilities. We need to work on basis of how can we get the best value of our instrumentation assuming lowest features on the backend side.
It also helps better tagging/logging practice, since having those advanced regex based search for instance, could lead to jamming all values in one tag and ending up with a worse tracing + getting locked into single backend.